stl/inc/yvals_core.h

```diff
@@ -952,7 +952,13 @@
 #define _CXX20_DEPRECATE_OLD_SHARED_PTR_ATOMIC_SUPPORT
 #endif // ^^^ warning disabled ^^^
 
-// next warning number: STL4030
+#if defined(_ITERATOR_DEBUG_ARRAY_OVERLOADS) && !defined(_SILENCE_ITERATOR_DEBUG_ARRAY_OVERLOADS_DEPRECATION_WARNING)

```

Comment:
I'm not sure this one needs a "silence" warning considering it can be silenced by removing the other define?

Reply:
I thought about projects that inherit some build flags from sub-projects.

Reply:
Maybe, but I've literally never seen someone turn this on :)

Reply:
Dont you care about projects with heterogeneous build flags *ducks away*

Reply:
I would care if I had ever seen someone even mention that this thing exists. (That we think the probability of someone noticing this removal is close to 0 is one of the primary reasons for removal)

I don't feel strongly here :)

Reply:
I agree with Billy - IDAO is extremely obscure (we haven't documented or advertised it), and if we thought there was any significant usage, we wouldn't be removing it. So, I don't think we need a "this option has been removed" message.

(Also, deprecation messages are different from removal messages. We emit deprecation messages when something is still available, but discouraged by either the Standard or Microsoft. This is an outright removal of IDAO.)

Reply:
Removed ;)